src/NServiceBus.Core/Hosting/HostingComponent.Configuration.cs

```diff
@@ -87,7 +75,7 @@ public void AddInstaller(Func<string, Task> installer)
 
             public string EndpointName { get; }
 
-            public IConfigureComponents Container { get; }
+            public IServiceCollection Container { get; }
```

Comment:
Should we rename `Container` to `Services`, it's not really a container anymore.